docs/library/machine.rst

```diff
@@ -166,6 +166,24 @@ Power related functions
 
    Availability: ESP32, WiPy.
 
+.. function:: wake_ext1_pins()
```

Comment:
The `machine` module, as much as possible, should contain port-agnostic functions. So the options here would be:

1. Move these functions to the `esp32` module, as they're port specific.
2. Make a generic `wake_pins()` function and document it as only available currently on the ESP32 port. This creates the possibility that other ports can support it in the future.

I don't know if returning only the EXT1 wake pins as a `wake_pins()` result would be too misleading or not, but I assume the function could be extended to include EXT0 pins if there were any?